src/constants/tokens.ts

```diff
@@ -374,6 +437,19 @@ function getCeloNativeCurrency(chainId: number) {
   }
 }
 
+export function isBoba(chainId: number): chainId is SupportedChainId.BOBA {
+  return chainId === SupportedChainId.BOBA
+}
+
+function getBobaNativeCurrency(chainId: number) {
+  switch (chainId) {
+    case SupportedChainId.BOBA:
+      return BOBA_BOBA
```

Comment:
Boba's default native currency is $ETH, and $BOBA is the alt fee token
Is there any fee switching logic involved/required?

Reply:
hey @souradeep-das it´s ETH ? not wETH? in that case i already update the token address , i´ll be reviewing if there exist a fee-switching review

Reply:
awesome, yes! the native currency is ETH (0x4200000000000000000000000000000000000006)